substrate/client/network/src/litep2p/service.rs

```diff
@@ -267,6 +273,10 @@ impl NetworkSigner for Litep2pNetworkService {
 }
 
 impl NetworkDHTProvider for Litep2pNetworkService {
+	fn find_closest_peers(&self, target: PeerId) {
+		let _ = self.cmd_tx.unbounded_send(NetworkServiceCommand::FindClosestPeers { target });
+	}
+
 	fn get_value(&self, key: &KademliaKey) {
```

Comment:
nit: Should we change this to `key: KademliaKey`, to make the clone explicit on the caller side or avoid it entirely?